client/src/components/AlumniMentorship.js

```diff
@@ -600,6 +624,21 @@ class RequestCards extends Component {
         this.setState({display: display})
     }
 
+    getActionItemsInput(selection) {
+        this.setState({
+            existingActionItems: selection.old,
+            newActionItems: selection.new,
+        });
+    }
+
+    async submitActionItems() {
+        let requests = await makeCall({
+            requestId: this.state.requestDetails._id,
+            actionItems: this.state.existingActionItems
+        }, `/request/actionItems/${this.props.userId}/${this.props.timeOffset}`, 'patch')
```

Comment:
Can we add a `swal` modal for success/failure? examples can be found in any of the *UpdateModals